contract/package.json

```diff
@@ -13,21 +13,24 @@
     "start": "yarn docker:make clean start-contract print-key",
     "build": "exit 0",
     "test": "ava --verbose",
-    "lint": "eslint '**/*.{js,jsx}'",
-    "lint-fix": "eslint --fix '**/*.{js,jsx}'",
-    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.{js,jsx}'",
-    "lint-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.{js,jsx}'"
+    "lint": "eslint '**/*.{js,ts}'",
+    "lint-fix": "eslint --fix '**/*.{js,ts}'",
+    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.js'",
+    "lint-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.js'",
+    "typecheck": "tsc"
```

Comment:
not critical:

elsewhere this is called `lint:types` and we have `lint:eslint` and `lint` does both.

see the PR linked from https://github.com/Agoric/agoric-sdk/discussions/8274#discussioncomment-7731577

Reply:
Agree, didn't want to change `lint-*` as a part of this, but agree it should standardized.

Reply:
see [3ce6458](https://github.com/Agoric/dapp-offer-up/pull/26/commits/3ce6458d29538bf40cfaa3183c09bfaa1f260ebe)